src/main/java/uk/gov/companieshouse/web/accounts/service/notehandler/smallfull/CurrentAssetsInvestmentsHandler.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.companieshouse.web.accounts.service.notehandler.smallfull;
+
+import org.apache.commons.lang.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.handler.Executor;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.model.accounts.smallfull.currentassetsinvestments.CurrentAssetsInvestmentsApi;
+import uk.gov.companieshouse.web.accounts.enumeration.NoteType;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.service.notehandler.NoteResourceHandler;
+import uk.gov.companieshouse.web.accounts.service.smallfull.SmallFullService;
+
+@Component
+public class CurrentAssetsInvestmentsHandler implements
+        NoteResourceHandler<CurrentAssetsInvestmentsApi> {
+
+    @Autowired
+    private SmallFullService smallFullService;
+
+    private static final UriTemplate STOCKS_URI =
```

Comment:
Should STOCKS_URI be called something like CURRENT_ASSETS_INVESTMENTS_URI for consistency?

Reply:
Good catch! Yes it should - renamed